sdk/spring-sdk/src/main/scala/kalix/springsdk/impl/SpringSdkMessageCodec.scala

```diff
@@ -54,59 +61,3 @@ private[springsdk] class SpringSdkMessageCodec extends MessageCodec {
   override def decodeMessage(value: ScalaPbAny): Any = value
 
 }
-
-private[springsdk] object SpringSdkMessageCodec {
-
-  /**
-   * Used in to compute cache value if absent. This method will try to scan the type hierarchy from the passed
-   * `messageClass` to for either an explicit JsonTypeName annotation or a JsonSubTypes.
-   *
-   * In the absence of any annotation from the JsonTypeInfo family, it will fallback to use the FQCN as a type hint.
-   */
-  private def findTypeHint(messageClass: Class[_]): String = {
```

Comment:
There is no reason for us to keep this complexity. If we keep it we'll need to document that users can choose between our own `@TypeName` or Jackson's `@JsonSubTypes` and `@JsonTypeName`. 

Previously it was important to have `@JsonSubTypes`, but since https://github.com/lightbend/kalix-jvm-sdk/pull/1123, it won't help the users. They do need to define individual handler per event.